### PR TITLE
Minimal doc for elastic search

### PR DESCRIPTION
**Pull request by Kyrremann**

### CODE REVIEW

File: docs/persistence/elastic-search.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Elastic Search
+
+!!! Info
+    This feature is only available in GCP clusters.
+
+The NAIS platform offer Elastic Search via Aiven.
```
Comment (chinatsu) [inline docs/persistence/elastic-search.md:6]: offers*

File: docs/persistence/elastic-search.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Elastic Search
+
+!!! Info
+    This feature is only available in GCP clusters.
```
Comment (chinatsu) [inline docs/persistence/elastic-search.md:4]: maybe point readers straight to the document? e.g. `[GCP clusters](../clusters/gcp/README.md)` here?

File: docs/persistence/elastic-search.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+## Get your own
+As there are few teams that need an Elastic Search instance we use a IaC-repo to provision each instance.
+Head over to [aiven-iac](https://github.com/navikt/aiven-iac#elastic-search) to learn how to get your own instance.
+
+### Username and password
+For now we are manually distributing the username and password for each instances.
+This user will be an admin user, which use can use to create the specific read/write users that you need.
+
+## Access from Nais-app
+If you need access from an application, use the following [nais.yaml-reference](https://doc.nais.io/nais-application/nais.yaml/reference/#specelasticinstance).
```
Comment (chinatsu) [inline docs/persistence/elastic-search.md:17]: use relative paths when pointing to an internal document
`[nais.yaml-reference](../nais-application/nais.yaml/reference.md#specelasticinstance)`

File: docs/persistence/elastic-search.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    This feature is only available in GCP clusters.
+
+The NAIS platform offer Elastic Search via Aiven.
+
+## Get your own
+As there are few teams that need an Elastic Search instance we use a IaC-repo to provision each instance.
+Head over to [aiven-iac](https://github.com/navikt/aiven-iac#elastic-search) to learn how to get your own instance.
+
+### Username and password
+For now we are manually distributing the username and password for each instances.
```
Comment (chinatsu) [inline docs/persistence/elastic-search.md:13]: for each instance*

File: docs/persistence/elastic-search.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+This user will be an admin user, which use can use to create the specific read/write users that you need.
```
Comment (chinatsu) [inline docs/persistence/elastic-search.md:14]: `This user will be an admin user which in turn can be used to create read/write users.` perhaps? at least `..., which can be used to create ...`

**Review by chinatsu — CHANGES_REQUESTED** [review]

File: docs/persistence/elastic-search.md (hunk 3 above)
Comment (Kyrremann) [reply docs/persistence/elastic-search.md:17]: Så klart!

File: docs/persistence/elastic-search.md (hunk 2 above)
Comment (Kyrremann) [reply docs/persistence/elastic-search.md:4]: Sure, kopierte den fra en annen side for å ha de litt like.

**Pull request merged**